zk/stages/stage_sequence_execute_utils.go

```diff
@@ -498,3 +499,69 @@ func (bdc *BlockDataChecker) AddTransactionData(txL2Data []byte) bool {
 
 	return false
 }
+
+type ResequenceBatchJob struct {
```

Comment:
Just a consideration, if you like it better, but doesn't really matter.

```suggestion
type ResequenceBatchTracker struct {
```

Reply:
I'd prefer to use Job, because it can represent both the job and the job progress :D